pkgs/tools/security/onlykey-agent/default.nix

```diff
@@ -52,7 +52,7 @@ python3Packages.buildPythonApplication rec {
     sha256 = "sha256-SbGb7CjcD7cFPvASZtip56B4uxRiFKZBvbsf6sb8fds=";
   };
 
-  propagatedBuildInputs = with python3Packages; [ lib-agent onlykey-cli setuptools ];
+  propagatedBuildInputs = with python3Packages; [ lib-agent onlykey-cli setuptools cython ];
```

Comment:
Since `cython` is a build time dependency, it should go to `nativeBuildInputs`.

Reply:
Hi @natsukium, adding `nativeBuildInputs = with python3Packages; [ cython ];` in either the lib-agent or onlykey-agent packages set sections fails to resolve the issue. 
`nativeBuildInputs` doesn't seem to work to stop `onlykey-agent` from crashing from a missing `cython` dependency [(stacktrace here)](https://github.com/NixOS/nixpkgs/pull/325182#issue-2393901208).

Either way, I think moving it to lib-agent's `propagatedBuildInputs` section makes more sense since based on the stack trace provided it would seem as if it's `libagent` that is depending on `cython` at runtime.

I'll make that commit and push it up.

Reply:
Thanks for the details.
Sorry I didn't look at the error log carefully, but I think your approach is reasonable.